### PR TITLE
Prince newbranch

### PR DESCRIPTION
**Pull request by princegeorge121**

### CODE REVIEW

File: client/mycart/src/Types/Order.Create.dto.d.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+    Customer: string;
+
+    DeliveryAddress: string;
+
+    PaymentStatus: string;
+
+    OrderTime: Date;
+
+    TotalPrice: number;
```
Comment (mishelshaji) [inline client/mycart/src/Types/Order.Create.dto.d.ts:13]: Follow variable naming conventions.

File: client/mycart/src/Types/Order.View.dto.d.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    Customer: string;
+
+    DeliveryAddress: string;
+
+    PaymentStatus: string;
+
+    OrderTime: Date;
+
+    TotalPrice: number;
```
Comment (mishelshaji) [inline client/mycart/src/Types/Order.View.dto.d.ts:15]: Follow naming conventions.

File: client/mycart/src/app/admin/admin-routing.module.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ const routes: Routes = [
       { path: 'products/create', component: AddProductComponent },
       { path: 'feedback/view', component: ViewFeedbackComponent },
       { path: 'products/update/:id', component: UpdateProductComponent },
-      { path: 'products', component: ProductsComponent }
+      { path: 'products', component: ProductsComponent },
       { path: 'categories/create', component: AddCategoriesComponent },
       { path: 'products/update/id', component: UpdateProductComponent }
```
Comment (mishelshaji) [inline client/mycart/src/app/admin/admin-routing.module.ts:20]: Convert id as route parameter.

File: client/mycart/src/app/services/order.service.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+@Injectable({
+  providedIn: 'root'
+})
+export class OrderService {
+  url = "https://localhost:7191/api/customer/order";
+
+  constructor(private http :HttpClient){}
+  
+  getAll(){
```
Comment (mishelshaji) [inline client/mycart/src/app/services/order.service.ts:12]: Format the code.

File: client/mycart/src/app/services/order.service.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    return this.http.get<OrderViewDto[]>(this.url);
+  }
+
+  postOrders(model : OrderCreateDto){
```
Comment (mishelshaji) [inline client/mycart/src/app/services/order.service.ts:16]: Format the code.

File: client/mycart/src/app/services/order.service.ts [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    return this.http.post<CategoryViewDto>(this.url, model);
+  }
+
+  delete(id : number){
```
Comment (mishelshaji) [inline client/mycart/src/app/services/order.service.ts:20]: Format the code.

File: server/MyCart/MyCart.Services/Services/OrderServices.cs [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+            Order order = new()
+            {
+                CustomerId = dto.CustomerId,
+                DeliveryAddress = dto.DeliveryAddress,
+                OrderTime = dto.OrderTime,
+                PaymentStatus = dto.PaymentStatus,
+                TotalPrice = dto.TotalPrice
+            };
+
+            // Insert and saves data into Database
```
Comment (mishelshaji) [inline server/MyCart/MyCart.Services/Services/OrderServices.cs:55]: Remove the comment.

File: client/mycart/src/Types/Order.Create.dto.d.ts (hunk 1 above)
Comment (princegeorge121) [reply client/mycart/src/Types/Order.Create.dto.d.ts:13]: Done.

File: client/mycart/src/Types/Order.View.dto.d.ts (hunk 2 above)
Comment (princegeorge121) [reply client/mycart/src/Types/Order.View.dto.d.ts:15]: Done.

File: server/MyCart/MyCart.Services/Services/OrderServices.cs (hunk 7 above)
Comment (princegeorge121) [reply server/MyCart/MyCart.Services/Services/OrderServices.cs:55]: Done.

File: client/mycart/src/app/admin/admin-routing.module.ts (hunk 3 above)
Comment (princegeorge121) [reply client/mycart/src/app/admin/admin-routing.module.ts:20]: Done.

File: client/mycart/src/app/services/order.service.ts (hunk 4 above)
Comment (princegeorge121) [reply client/mycart/src/app/services/order.service.ts:12]: Done.

File: client/mycart/src/app/services/order.service.ts (hunk 6 above)
Comment (princegeorge121) [reply client/mycart/src/app/services/order.service.ts:20]: Done.

File: client/mycart/src/app/services/order.service.ts (hunk 5 above)
Comment (princegeorge121) [reply client/mycart/src/app/services/order.service.ts:16]: Done.

**Review by mishelshaji — APPROVED** [review]

**Pull request closed without merging**